cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=255)
+    description = serializers.CharField(required=True)

```

Comment:
it's `required=True` by default

Reply:
You could remove `required=True` flag here and in the other fields as well since it is a default option